clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fmri.py

```diff
@@ -130,21 +129,33 @@ def _compute_fmri_path(
     fmri_dfs_list = []
 
     # Loading needed .csv files
-    adni_merge = load_clinical_csv(csv_dir, "ADNIMERGE")
-
-    mayo_mri_qc = load_clinical_csv(csv_dir, "MAYOADIRL_MRI_IMAGEQC_12_08_15")
-
+    # adni_merge_path = path.join(csv_dir, "ADNIMERGE.csv")
```

Comment:
to be deleted ?

Reply:
Agreed

Reply:
Line 58 should be kept instead of the lines below. I had written a different method for file imports since I was having an issue with the CSVs, however, this solution may not be the same for everyone, depending on how the CSVs are formatted for each person upon downloaded (maybe it was operating system dependent, I'm not sure)

Line 71 n_procs can be kept with paths_to_bids

Keep line 114

Keep lines 133 to 136